src/components/common/YourCompany/CorrectName/CorrectNameRequest.vue

```diff
@@ -214,43 +215,46 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
             'OK'
           )
         } else {
-          this.parseNameRequest(nr)
+          this.storeNameRequest(nr)
           // Set our entity type, if it's a conversion request
           if (nr.request_action_cd === NrRequestActionCodes.CONVERSION) {
-            this.setEntityType(nr.legalType)
+            this.setEntityType(nr.legalType as any)
             this.setEntityTypeChangedByName(true)
           }
-          this.emitIsSaved(true)
+          this.emitSaved(true)
         }
       } catch {
-        // "validateNameRequest" handles its own errors
+        // "fetchValidateNameRequest" handles its own errors
         // Inform parent process is complete
-        this.emitIsSaved()
+        this.emitSaved()
       }
     }
   }
 
   /* Checks name request type or if it's an invalid conversion name request. */
-  isNameRequestInvalid (nr: NrResponseIF): boolean {
-    const isNameEntityTypeDifferent = this.getEntityType !== nr.legalType
+  isNameRequestInvalid (nr: NameRequestIF): boolean {
+    const isNameEntityTypeDifferent = (this.getEntityType !== nr.legalType as any)
```

Comment:
Why are we casting as any here? Aren't they both of `CorpTypeCd`?

Reply:
They sure are the same, but VS Code complains about it:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/18380e7c-8910-4ffb-84d9-0cb16c94185c)

As mentioned elsewhere, I think it's because VS Code (or the Vue/Typescript parser/compiler in general) isn't recognizing imported types as the same. We might need to create a new d.ts file, but that's a problem for another day. In the meantime, I've changed all "as unknown as xxx" to "as any" for easier search-and-destroy later.

Reply:
> They sure are the same, but VS Code complains about it:
> 
> ![image](https://private-user-images.githubusercontent.com/10002889/324119979-18380e7c-8910-4ffb-84d9-0cb16c94185c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1NjIyMTAsIm5iZiI6MTcxMzU2MTkxMCwicGF0aCI6Ii8xMDAwMjg4OS8zMjQxMTk5NzktMTgzODBlN2MtODkxMC00ZmZiLTg0ZDktMGNiMTZjOTQxODVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE5VDIxMjUxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZjZWM4OWFlMzI3ZDA1NjM5MmRjNmVmZTA4ZWQwMWU5MGUwNGY2Njg3MDBlNTRhMWUyMTUwYjUyYjlmNzYwZmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JhpwpLjNhqsdr53DPx1lkRcifm5uLo44G9jxkHTT0XA)
> 

Ah, I see. I remember this VS Code complaint well. I stumbled upon it when I was the Pinia upgrade back then I believe.

OK Sev, we can deal with it later. No need to bug yourself with it now. A problem for another day.